docs/user-guide/admin-cli.md

```diff
@@ -9,14 +10,341 @@ by the Docker user via the Docker CLI. However, the Docker CLI is limited in wha
+  --name NAME           Tenant name
+  --datastore DATASTORE
+                        Datastore name
+```
+### List
+
+List existing tenants, the datastores tenants have access to and the VMs assigned.
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+Uuid                                  Name     Description  Default_datastore  VM_list
+------------------------------------  -------  -----------  -----------------  ---------------------------
+dc17b3b1-1f4d-4f8b-a6f4-3dba064ba88f  tenant1               default_ds         photon.vsan.1,photon.vsan.2
+5afa4767-ecd8-489e-9d97-5c46ae50d3fc  DevAppX               default_ds         ubuntu
+```
+#### Help
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+Uuid                                  Name     Description  Default_datastore  VM_list
+------------------------------------  -------  -----------  -----------------  ---------------------------
+dc17b3b1-1f4d-4f8b-a6f4-3dba064ba88f  tenant1               default_ds         photon.vsan.1,photon.vsan.2
+5afa4767-ecd8-489e-9d97-5c46ae50d3fc  DevAppX               default_ds         ubuntu
+
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls -h
+usage: vmdkops_admin.py tenant ls [-h]
+
+optional arguments:
+  -h, --help  show this help message and exit
+```
+
+### Virtual Machine
+#### Add
+Add a VM to a tenant. A VM can only access the datastores for the tenant it is assigned to.
+VMs can be assigned to only one tenant at a time.
+#### List
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm ls --name tenant1
+Uuid                                  Name
+------------------------------------  -------------
+564d6f23-eabb-1b63-c79d-2086cca704d5  photon.vsan.2
+
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm add --name tenant1 --vm-list photon.vsan.1
+tenant vm add succeeded
+```
+#### Remove
+Remove a VM from a tenant's list of VMs. VM will no longer be able to access the volumes created for the tenant.
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm rm --name tenant1 --vm-list photon.vsan.1
+tenant vm rm succeeded
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm ls --name tenant1
+Uuid                                  Name
+------------------------------------  -------------
+564d6f23-eabb-1b63-c79d-2086cca704d5  photon.vsan.2
+```
+#### Help
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm -h
+usage: vmdkops_admin.py tenant vm [-h] {rm,add,ls} ...
+
+positional arguments:
+  {rm,add,ls}
+    rm         Remove VM(s) from a tenant
+    add        Add a VM(s) to a tenant
+    ls         list VMs in a tenant
+
+optional arguments:
+  -h, --help   show this help message and exit
 ```
+### Remove
+
+Remove a tenant, optionally all volumes for a tenant can be removed as well.
```

Comment:
The "tenant rm" command will print "tenant rm succeeded" if command succeeded.  Please include that in the command example.